### ISSUE TITLE
Remove the See Also Section in a name entry

### ISSUE THREAD

**Issue by dadepo** [Yorubaname/yorubaname-website#29]
The "See Also" in the name entry view appears not to have any use. 

It should be removed.

**Comment by kolatubosun** [comment]
I'd say it should NOT be removed. The problem I had with it before was that there was no provision for it on the back end, even though it shows on the front end, so I couldn't use it.

I've found a couple of uses for it now, but it needs to be provided for on the dashboard. The original reason for its creation was to be able to direct users who come into a name (say, "Olubunmi") to see other variants that aren't the normal variants. Normal variants for "olubunmi" will be "Bunmi" or "Oluwabunmi". The "See also" for this name will be "Fabunmi", which is a variant of Bunmi. Just not a direct variant of "Olubunmi".

A better example is "Fajuyi". Normal variant of this is "Fajuyigbe" or "Ifajuyigbe". See also would be "Olujuyigbe" and "Adejuyigbe".

**Comment by Zifah** [comment]
@kolatubosun Based on your last comment, it seems like this one should be closed. Is that correct?

**Comment by kolatubosun** [comment]
Has it been fixed? To be honest, I think what we need for this space is an AI-generated result for names related to the entry in some relevant way. Can we get that?

**Comment by Zifah** [comment]
No, it hasn't. I commented because your comment indicated that removing that section is not the way to go.

Of course, it's possible, but it's not trivial.
